source/includes/harvest/_candidates.md

```diff
@@ -1167,7 +1167,7 @@ curl -X POST 'https://harvest.greenhouse.io/v1/prospects'
 }
 ```
 
-Create a new prospect. The difference between a prospect and a candidate is that a prospect can be on no jobs or many jobs. A prospect application cannot be added to a job stage. When a prospect is ready to be added to a job stage, they can be converted to a candidate in Greenhouse. Alternatively, you can add a candidate application to a prospect's profile by using the [Add Candidate Application] (#post-add-candidate-application) endpoint. 
+Create a new prospect. The difference between a prospect and a candidate is that a prospect can be on no jobs or many jobs. A prospect application cannot be added to a job stage. When a prospect is ready to be added to a job stage, they can be converted to a candidate in Greenhouse. Alternatively, you can add a candidate application to a prospect's profile by using the [Add Candidate Application] (#post-add-candidate-application) endpoint. **CRM is required to access this endpoint**
```

Comment:
Let's link out to the Marketing page once it exists for people that don't know what CRM is.

Reply:
@MarkyMarkMcDonald @davidhan527 - does this exist as of today?

Reply:
@tdphillipsjr this does exist but we need to change this copy a bit. Will update and let you know. Whats the process for merging these PR's after it's been approved?